lldb/packages/Python/lldbsuite/test/tools/lldb-dap/lldbdap_testcase.py

```diff
@@ -155,15 +152,13 @@ def assertCapabilityIsNotSet(self, key: str, msg: Optional[str] = None) -> None:
         if key in self.dap_server.capabilities:
             self.assertEqual(self.dap_server.capabilities[key], False, msg)
 
-    def verify_breakpoint_hit(
-        self, breakpoint_ids: List[Union[int, str]], timeout: float = DEFAULT_TIMEOUT
-    ):
+    def verify_breakpoint_hit(self, breakpoint_ids: List[Union[int, str]]):
```

Comment:
This reminds me, I want to revisit breakpoint id handling.

We're inconsistent in handling breakpoint ids. Some APIs take them as strings, some convert them to strings and some take them as ints. They're an `int` in the spec, but I'm not sure why we're converting them into strings...